src/sagemaker/workflow/steps.py

```diff
@@ -93,6 +93,26 @@ def ref(self) -> Dict[str, str]:
         return {"Name": self.name}
 
 
+@attr.s
+class CacheConfig:
+    """Step to cache pipeline workflow.
+
+    Attributes:
+        enable_caching (bool): To enable step caching. Off by default.
+        expire_after (str): If step caching is enabled, a timeout also needs to defined.
```

Comment:
This should be enforced, if declared. Could you include logic to enforce this?

Reply:
expire after is optional